google-cloud-spanner/src/main/java/com/google/cloud/spanner/TransactionRunnerImpl.java

```diff
@@ -1079,7 +1092,7 @@ public TransactionRunner allowNestedTransaction() {
   TransactionRunnerImpl(SessionImpl session, TransactionOption... options) {
     this.session = session;
     this.options = Options.fromTransactionOptions(options);
-    this.txn = session.newTransaction(this.options);
+    this.txn = session.newTransaction(this.options, null);
```

Comment:
nit (here and elsewhere): It is good practice to add a `/* prevTransactionId = */` style comment when you are passing in an argument like this that is otherwise hard to understand what it represents.

Reply:
Done. Thanks for the suggestion